Amplify/Core/Configuration/Internal/Amplify+Reset.swift

```diff
@@ -32,14 +32,26 @@ extension Amplify {
                 reset(Auth, in: group) { group.leave() }
             case .dataStore:
                 reset(DataStore, in: group) { group.leave() }
-            case .hub:
-                reset(Hub, in: group) { group.leave() }
-            case .logging:
-                reset(Logging, in: group) { group.leave() }
             case .storage:
                 reset(Storage, in: group) { group.leave() }
             case .predictions:
                 reset(Predictions, in: group) { group.leave() }
+            case .hub, .logging:
+                // should be waited until other finish resetting
+                break
```

Comment:
this class has a lot of hard coding logging and hub categories, maybe you can isolate these two categories that you can reuse in the rest of the code, like
```
let resetLastCategories = [CategoryType.hub, .logging]
```
then you can filter before line 25 by `let resetFirstCategories = CategoryType.allCases.filter { !resetLastCategories.contains($0) }`

With this, would you be able to avoid empty switch cases, similar to code like this? 
```
for categoryType in resetFirstCategories {
    reset(categoryType.category, in: group) { group.leave() }
}

group.wait()

for categoryType in resetLastCategories {
    reset(categoryType.category, in: group) { group.leave() }
}
group.wait()


```